test/create_exercise_test.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:test/test.dart';
+import '../bin/create_exercise.dart';
+
+void main() {
+  group('create-exercise tool', () {
+    group('helpers', () {
+      test('words()', () {
```

Comment:
I recommend separating out each test for words() (and the other function tests) so developers know what is being tested. If they can be grouped together like testing valid words, that's fine. But some separation could be useful

Reply:
I'm not sure I understand how you'd like to separate this.

Reply:
I was suggesting to have a group of tests for each helper function.

For example, word() may have 4-5 separate tests within a single group.

When a test fails we should know what functionality broken, not which function stops working.

Currently, if the test fails, we may know when the first expect function failed, but we won't know if the other expect functions were valid. Therefore, I believe it would be best for each test to have one expect so if something breaks we have clearer picture of what is working and what isn't.

Reply:
Oh, I see what you mean! That actually makes more sense, yes.